File: src/ml/Models/StatisticsClassification.py
class StatisticsClassification:
    
    def __init__(
        self,
        Accuracy         = 0,
        BalancedAccuracy = 0,
        Precision        = 0,
        Recall           = 0,
        F1Score          = 0,
        HammingLoss      = 0,
        CrossEntropyLoss = 0,
        ConfusionMatrix  = []
        ) -> None:
        
        self.Accuracy         = str(float(Accuracy))
        self.BalancedAccuracy = str(float(BalancedAccuracy))
        self.Precision        = str(float(Precision))
        self.Recall           = str(float(Recall))
        self.F1Score          = str(float(F1Score))
        self.HammingLoss      = str(float(HammingLoss))
        self.CrossEntropyLoss = str(float(CrossEntropyLoss))
        self.ConfusionMatrix  = [[int(i) for i in row] for row in ConfusionMatrix]
    
    

File: src/backend/dotNet/dotNet/Models/StatisticsClassification.cs
namespace dotNet.Models {
    public class StatisticsClassification {
        public StatisticsClassification(
            float accuracy, 
            float balancedAccuracy, 
            float precision, 
            float recall, 
            float f1Score, 
            float hammingLoss, 
            float crossEntropyLoss, 
            int[][]? confusionMatrix
            ) {
            Accuracy = accuracy;
            BalancedAccuracy = balancedAccuracy;
            Precision = precision;
            Recall = recall;
            F1Score = f1Score;
            HammingLoss = hammingLoss;
            CrossEntropyLoss = crossEntropyLoss;
            ConfusionMatrix = confusionMatrix;
        }

        public float Accuracy { get; set; }
        public float BalancedAccuracy { get; set; }
        public float Precision { get; set; }
        public float Recall { get; set; }
        public float F1Score { get; set; }
        public float HammingLoss { get; set; }
        public float CrossEntropyLoss { get; set; }
        public int[][]? ConfusionMatrix { get; set; }

    }
}
